x-pack/plugins/security_solution/cypress/e2e/detection_alerts/changing_alert_status.cy.ts

```diff
@@ -360,6 +361,7 @@ describe('Changing alert status', () => {
           );
 
           closeAlerts();
+          refreshAlertPageFilter();
```

Comment:
We should remove this piece of code.

When performing any action on the alerts, the table should be automatically refreshed. The need of refreshing the page to make sure the new number of alerts is correctly displayed is a sign that something is not working fine and needs to be fixed.

Reply:
Yes, I agree.  It is done now.